packages/utils/odsp-doclib-utils/src/odspRequest.ts

```diff
@@ -13,7 +13,8 @@ export async function getAsync(
     url: string,
     authRequestInfo: IOdspAuthRequestInfo,
 ): Promise<Response> {
```

Comment:
Shouldn't this update to `Promise<any>`?

Also, next time please provide an initial comment/description on the PR and on the commit for context for others who look at this PR or commit in the future.

Reply:
@jatgarg

Reply:
In driver too, we directly cast this to Response as we expect it to be a Response object.
Yes sure for the description thing. It was small so I thought the heading is sufficient.